### DISCUSSION TITLE
Windows 11 detached commands on drive other than C:

### DISCUSSION THREAD

**Discussion by quikbeam** [LizardByte/community#123] [Sunshine] [answered]
I have a few games installed on a backup E: drive.  When I try a detach command like "E:\Games\Nioh 2 - Complete Edition\nioh2.exe".  The game launches and closes.  Anthing on the C: drive works without any issues.  I've tried setting the working directoy to "E:\Games\Nioh 2 - Complete Edition" and tried creating a symlink (mklink /d "C:\Games" "E:\Games") and chaging the drive to C:

I wanted to avoid creating .bat files for all my games.  Is a drive other than C: supported or am I missing something?

**Comment by ReenigneArcher** [comment, marked as answer]
What version are you using? Try nightly please... https://github.com/LizardByte/Sunshine/releases/tag/nightly-dev

**Reply by quikbeam** [reply]
Nightly doesn't have this issue and I noticed HDR is also working!
